test/functional/cursor.test.js

```diff
@@ -3901,6 +3901,49 @@ describe('Cursor', function () {
     });
   });
 
+  // NODE-2035
+  it('should propagate error when exceptions are thrown from an awaited forEach call', function () {
```

Comment:
is there any way we can simplify this test? I realize that the other tests in this file use similar syntax, but the better practice would be to outsource shared setup into a beforeEach (establishing the client, creating the collection) and the cleanup into an afterEach (closing the client and cursor connections) instead of the defer syntax here

Reply:
Would it be acceptable for now to create a new describe block with a `beforeEach` setup block and `afterEach` teardown block just for this test at the moment?